src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -120,13 +132,15 @@ function ReportActionItemMessageEdit(props) {
         // For mobile Safari, updating the selection prop on an unfocused input will cause it to automatically gain focus
         // and subsequent programmatic focus shifts (e.g., modal focus trap) to show the blue frame (:focus-visible style),
         // so we need to ensure that it is only updated after focus.
-        setDraft((prevDraft) => {
-            setSelection({
-                start: prevDraft.length,
-                end: prevDraft.length,
+        if (isMobileSafari) {
+            setDraft((prevDraft) => {
+                setSelection({
+                    start: prevDraft.length,
+                    end: prevDraft.length,
+                });
+                return prevDraft;
             });
-            return prevDraft;
-        });
+        }
```

Comment:
What you think about the early return there? Just a styling suggestion, fell free to discuss 

```suggestion
        if (!isMobileSafari) {
            return;
        }
        setDraft((prevDraft) => {
            setSelection({
                start: prevDraft.length,
                end: prevDraft.length,
            });
            return prevDraft;
        });
```

Reply:
We can't as it will break the `useEffect's` flow.

Reply:
You're right, my bad 😅